### PR TITLE
[habpanel] Documentation fixes

### PR DESCRIPTION
**Pull request by osama-salah**
Changed "Get widgets from the openHAB community"  to "Get more..."

### CODE REVIEW

**Comment by stefan-hoehn** [comment]
@florian-h05 this is pretty old but a valid addition to the text we should merge. I would even add this image because it makes it easier to find for new users 

image

There is also a wrong wording which we should correct at the same time

wrong

> The list of custom widgets either via the dashboard designer (click/tap the gears icon in the Add Widget dropdown menu), or with the Manage button in the settings screen.

correct

The list of custom widgets can be found either via the dashboard designer (click/tap the gears icon in the Add Widget dropdown menu), or with the Manage button in the settings screen.

File: bundles/org.openhab.ui.habpanel/doc/habpanel.md [hunk 1]
```diff
@@ -262,7 +262,7 @@ For more information, please read the [community thread about this feature](http
 
 The list of custom widgets either via the dashboard designer (click/tap the gears icon in the _Add Widget_ dropdown menu), or with the _Manage_ button in the settings screen.
```
Comment (florian-h05) [inline bundles/org.openhab.ui.habpanel/doc/habpanel.md:263]:
```suggestion
The list of custom widgets can be found either via the dashboard designer (click/tap the gears icon in the _Add Widget_ dropdown menu), or with the _Manage_ button in the settings screen.
```

**Review by florian-h05 — APPROVED** [review]
LGTM, thanks!

**Comment by florian-h05** [comment]
> There is also a wrong wording which we should correct at the same time

I corrected it

**Pull request merged**